modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Resource.scala

```diff
@@ -19,7 +19,8 @@ object Resource {
   case object Gems extends Resource
   case object Altair extends Resource
   trait Instrument extends Resource
-  case object GMOS extends Instrument
+  case object GMOS_S extends Instrument
```

Comment:
This has changed a bit upstream, so you'd need to rebase. I defined GMOS-N to get the UI to work